composer/algorithms/low_precision_layernorm/low_precision_layernorm.py

```diff
@@ -143,7 +143,7 @@ def _to_LPLayerNorm(layer: torch.nn.Module, module_index: int) -> LPLayerNorm:
     lp_layernorm = LPLayerNorm(layer.normalized_shape, layer.eps, layer.elementwise_affine)
 
     with torch.no_grad():
-        if hasattr(layer, 'weight'):
+        if not hasattr(layer, 'weight'):
```

Comment:
```suggestion
        if layer.weight is None:  # pyright: ignore[reportUnnecessaryComparison]
```
matches check in rest of file